### PR TITLE
unit tests for recent news component

### PR DESCRIPTION
**Pull request by simon-rosengren**

### CODE REVIEW

**Comment by MoysisPap** [comment]
Good job!

**Review by MoysisPap — CHANGES_REQUESTED** [review]
The tests look good but maybe you could add a handle api error

File: src/components/newsSection/RecentNews.test.jsx
Comment (HifzaBatool) [inline src/components/newsSection/RecentNews.test.jsx:1]: Test Approved!

**Review by HifzaBatool — APPROVED** [review]
Test Approved!

**Review by vsafonova — CHANGES_REQUESTED** [review]
You did great work, but you could add a test for edge cases. What happens if the news items list is empty?

**Comment by simon-rosengren** [comment]
I will add tests for edge cases and API errors in another merge!

**Pull request merged**